src/cartridges/int_adyen_overlay/cartridge/scripts/adyenGetOpenInvoiceData.js

```diff
@@ -70,9 +70,9 @@ function getLineItems(args) {
       lineItemObject.id = id;
       lineItemObject.quantity = quantity;
       lineItemObject.taxCategory = 'None';
-      lineItemObject.taxPercentage = (
+      lineItemObject.taxPercentage = args.addTaxPercentage ? (
         new Number(vatPercentage) * 10000
```

Comment:
Could be NaN if vatPercentage is not defined.

Reply:
vatPercentage is declared as 0 [here](https://github.com/Adyen/adyen-salesforce-commerce-cloud/blob/develop/cartridges/int_adyen_overlay/cartridge/scripts/util/lineItemHelper.js#L53), so that should not be an issue.